### PR TITLE
Fix cmake , include the change of dtk, and use dwayland instead

### PR DESCRIPTION
**Pull request by Decodetalkers**
Log:

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -41,9 +41,8 @@ endif ()
 
 # Find the library
 find_package(PkgConfig REQUIRED)
-find_package(DtkWidget REQUIRED)
-find_package(DtkCore REQUIRED)
-find_package(Qt5 COMPONENTS Core Widgets DBus Test REQUIRED)
+find_package(Dtk COMPONENTS Widget Core)
```
Comment (BLumia) [inline CMakeLists.txt:44]:
```suggestion
find_package(Dtk COMPONENTS Widget Core REQUIRED)
```

**Review by BLumia — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (Decodetalkers) [reply CMakeLists.txt]: done

File: CMakeLists.txt [hunk 2]
```diff
@@ -248,7 +241,6 @@ target_include_directories(${UT_BIN_NAME} PUBLIC
 
 target_link_libraries(${UT_BIN_NAME} PRIVATE
     ${DtkWidget_LIBRARIES}
```
Comment (BLumia) [inline CMakeLists.txt:243]: 这个也可以去了吧？

**Pull request merged**